docs/doc_tutorial/10_requestdata.rst

```diff
@@ -1,5 +1,90 @@
 ========================================
-Request some json data
+Request JSON data
 ========================================
 
-soon...
\ No newline at end of file
+In this tutorial, we will use flares API to load some JSON data from an API and process it.
+
+Project setup
+--------------------
+
+Please refer to the "Hello World" tutorial on how to set up a basic project with flare.
+
+Using HTTPRequest
+--------------------
+
+Flare comes with a high level API to request data. The ``flare.network`` module contains a class ``HTTPRequest``, whose
+constructor takes six parameters:
+
+1. ``method``: The HTTP method to use for the request (i.e. ``GET``, ``POST``, ...)
+2. ``url``: The URL to request
+3. ``callbackSuccess`` (optional): A reference to the function which is to be called when the request succeeds (takes a response parameter)
+4. ``callbackFailure`` (optional): A reference to the function which is to be called when the request fails (take the parameters responseText and status)
+5. ``payload`` (optional): The body of the request, if one is to be sent
+6. ``content_type`` (optional): A value for a ``Content-Type`` header (e.g. ``application/json``)
+
+Using this constructor immediately sends the request.
+
+Handling the response
+--------------------
+
+In order to parse a JSON response in a success callback, simply use the default ``json`` functionality:
+
+.. code:: python
+
+    def successCallback(result):
+      data = json.loads(result)
+
+This will simply turn the response into a ``dict`` whose fields can then be accessed in the normal way.
```

Comment:
This must not be a `dict`. It could also be another [JSON decoded object](https://github.com/python/cpython/blob/a3d20bfee38c71df88d69064d47fe98a1d59e624/Lib/json/decoder.py#L259-L277).